.eslintrc

```diff
@@ -397,6 +397,9 @@
     "wrap-iife": [
       "error",
       "any"
-    ]
+    ],
+    // note you must disable the base rule as it can report incorrect errors
+    "dot-notation": "off",
```

Comment:
This is duplicate, this rule is defined at line 177. Please remove it from there.